### PR TITLE
create own portfolio

### PR DESCRIPTION
**Pull request by sangam-verma9**

### CODE REVIEW

File: Your Portfolio/sangam-verma9.html [hunk 1]
```diff
@@ -0,0 +1,995 @@
...
+  </div>
+
+
+
+<!-- DO NOT CHANGE THE CODE BELOW -->
+    <!-- JavaScript Library - Bootstrap JS -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
+</body>
+</html>
+Footer
```
Comment (isyneha) [inline Your Portfolio/sangam-verma9.html:981]: Remove these lines

File: Your Portfolio/sangam-verma9.html [hunk 2]
```diff
@@ -0,0 +1,995 @@
+Skip to content
```
Comment (isyneha) [inline Your Portfolio/sangam-verma9.html:1]: REmove Line No- 1 to 30

**Review by isyneha — CHANGES_REQUESTED** [review]
Please remove these extra lines from above and below.

File: Your Portfolio/sangam-verma9.html (hunk 2 above)
Comment (sangam-verma9) [reply Your Portfolio/sangam-verma9.html]: ok maam done

**Review by isyneha — APPROVED** [review]
LGTM, Approving

**Pull request merged**